web_src/js/modules/fetch.test.js

```diff
@@ -0,0 +1,14 @@
+import {test, expect} from 'vitest';
+import {GET, POST, PATCH, PUT, DELETE, isNetworkError} from './fetch.js';
+
+test('exports', () => {
+  expect(GET).toBeTruthy();
+  expect(POST).toBeTruthy();
+  expect(PATCH).toBeTruthy();
+  expect(PUT).toBeTruthy();
+  expect(DELETE).toBeTruthy();
+});
```

Comment:
This test seems a no-op

Reply:
It is there to satisfy the linter which would otherwise complain about unused functions. Could also just not defined the unused functions I guess.